.github/ISSUE_TEMPLATE/feature_requet.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Submit a feature request 
+title: ''
+labels: 'kind/feature-request'
+assignees: ''
+
+---
+
+**Is your feature request related to a use case or problem? Please describe.**
+
+
+**Describe the solution you'd like**
+
+
+**Describe alternatives/workarounds you've considered**
+
+
+**Additional context (e.g. screenshots)**
+
+
+**What is the urgency from your perspective for this issue? Is it blocking important work?**
+<!-- Please choose one and remove the others -->
+
+P1 - this should land no later than a week
```

Comment:
It seems like all of these priorities should be one less.  P0 = less than a week, etc.  That seems closer to how we rank things internally.

Reply:
I'm open to it. 
My thinking was that I'd use a single priority labelling for both bugs and features - and I just wanted to hold P0 for hotfixes which I thought would only be for bugs - I can hardly imagine the lack of a new feature being critical/harmful that needs a hotfix immediately (the class I would use for P0) while bugs can reach that level.

Reply:
Ah, that's a good point.  In the same vein, do we really have features that need to be done within a week?!?